### PR TITLE
Improve debugging of MSP code

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by haslinghuis** [comment]
@blckmn 

Please change debug setting here for GA:
image

**Comment by blckmn** [comment]
> @blckmn
> 
> Please change debug setting here for deprecated GA: image
> 
> Suggestion from Google: https://developers.google.com/analytics/devguides/migration/ua/analyticsjs-to-gtagjs

You are going to need to be a bit more descriptive of exactly what you are asking me to do. I am not familiar with this.

**Review by McGiverGim — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
